app/javascript/hyacinth_v1/components/projects/publish_targets/PublishTargetForm.js

```diff
@@ -16,9 +18,10 @@ import {
 } from '../../../graphql/publishTargets';
 import GraphQLErrors from '../../shared/GraphQLErrors';
 
+const publishTargetTypes = ['PRODUCTION', 'STAGING'];
```

Comment:
Just a general note that at some point in the future, it would be good if we could grab these values from the server side so we don't need to keep them in sync on the client side.  I'll ticket that.

Reply:
HYACINTH-618